corpora-api.yml

```diff
@@ -493,6 +497,131 @@ components:
+                type: string
+          links:
+            type: object
+            properties:
+              protocol:
+                type: array
+                items:
+                  type: string
+              other:
+                type: array
+                items:
+                  type: string
+              raw_data:
+                type: array
+                items:
+                  type: string
+              attestation:
+                type: array
+                items:
+                  type: string
+          datasets:
+            type: array
+            items:
+              type: object
+              properties:
+                id:
+                  $ref: '#/components/schemas/dataset_uuid'
+                assay:
+                  type: string
+                assay_ontology:
+                  type: strings
+                tissue:
+                  type: string
+                tissue_ontology:
+                  type: string
+                disease_state:
+                  type: string
+                disease_state_ontology:
+                  type: string
+                sex:
+                  type: string
+                ethnicity:
+                  type: string
+                ethnicity_ontology:
+                  type: string
+                organism:
+                  type: string
+                organism_ontology:
+                  type: string
+                title:
+                  type: string
+                source_data_location:
+                  type: string
+                revision:
+                  type: string
+                dataset_deployments:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                      environment:
+                        type: string
+                      link:
+                        type: string
+                dataset_assets:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                      fromat:
+                        type: string
+                      file_name:
+                        type: string
+                      type:
+                        type: string
+                        enum: [remix, original]
+                      s3_uri:
+                        type: string
+                contributors:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                      name:
+                        type: string
+                      email:
+                        type: string
+                      institution:
+                        type: string
+                dois:
```

Comment:
I think we want two separate DOIs here: a `preprint_doi` and a `publication_doi`.

Reply:
how are they differentiated?

Reply:
One project has up to two DOIs; the preprint and publication DOIs are different numbers thus designating different "journals" (technically the preprint will be in bioRxiv).

Reply:
How are they differentiated in the DB?